cms/djangoapps/contentstore/tasks.py

```diff
@@ -684,21 +684,31 @@ def validate_course_olx(courselike_key, course_dir, status):
         LOGGER.exception(f'{log_prefix}: CourseOlx Could not be validated')
         return olx_is_valid
 
+    log_errors = len(errorstore.errors) > 0
```

Comment:
```suggestion
       if errorstore.errors:
        log_errors_to_artifact(errorstore, status)
```

the redundant variable can be removed if not used elsewhere and in if condition  `errorstore.errors` can b e used directly no need to calculate `len`

Reply:
Fair enough 👍